lib/internal/modules/esm/module_job.js

```diff
@@ -59,25 +64,20 @@ class ModuleJob {
     // 'unhandled rejection' warnings.
     this.linked.catch(noop);
 
-    // instantiated == deep dependency jobs wrappers instantiated,
-    // module wrapper instantiated
+    // instantiated == deep dependency jobs wrappers are instantiated,
+    // and module wrapper is instantiated.
     this.instantiated = undefined;
   }
 
-  async instantiate() {
-    if (!this.instantiated) {
-      return this.instantiated = this._instantiate();
+  instantiate() {
```

Comment:
nit: I prefer to keep this sort of code async functions and `return await` in order to get a more convenient stack trace though in this case since this is in the loader it's probably better this way.

Reply:
`return` is already an await point, the stack traces still work afaik.

Reply:
I thought they didn't but I just checked and they do. I didn't know they fixed that.